log/sequencer.go

```diff
@@ -22,6 +23,11 @@ type Sequencer struct {
 	timeSource util.TimeSource
 	logStorage storage.LogStorage
 	keyManager crypto.KeyManager
+
+	// These parameters can be safely adjusted during operation
```

Comment:
Maybe "The following parameters...".

Do the mutable parameters need to be protected by a `sync.RWMutex` if they can potentially be written by different threads than are reading them?